build/CoreFoundation/dll/CoreFoundation.vcxproj

```diff
@@ -152,7 +152,7 @@
       <IgnoreAllDefaultLibraries>false</IgnoreAllDefaultLibraries>
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
       <ModuleDefinitionFile>CoreFoundation.def</ModuleDefinitionFile>
-      <AdditionalDependencies>mincore.lib;libxml2.lib;icudt.lib;icuin.lib;icuuc.lib;libdispatch.lib;icudata.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
CoreFoundation.dll 18.8mb -> 0.9mb

---
In reply to: [71389811](https://github.com/Microsoft/WinObjC/pull/668#discussion_r71389811) [](ancestors = 71389811)